docs/source/index.rst

```diff
@@ -23,10 +21,18 @@ This document introduces Olive and provides some examples to get you started.
    getstarted/installation
    getstarted/quickstart_examples
 
+.. toctree::
+   :maxdepth: 1
+   :caption: EXAMPLES
+
+   examples
+
 .. toctree::
    :maxdepth: 1
    :caption: TUTORIALS
 
+   tutorials/design
+   tutorials/options
```

Comment:
There are two kinds of tutorials and we should separate them in subsections
- Tutorials for Model developers (who are Olive users)
- Tutorials for Olive developers (who are Olive contributors)

Reply:
Currently only "How to add new Pass" is tutorial for contributors. I move this part to "Guide for contributors"